tests/recovery.py

```diff
@@ -536,8 +536,9 @@ def test_recover_service_from_files(
         os.path.dirname(os.path.realpath(__file__)), "testdata", directory
     )
 
-    old_common = os.path.join(service_dir, "common")
     new_common = infra.network.get_common_folder_name(args.workspace, args.label)
+    remove_tree(new_common)
```

Comment:
I think it'd be nicer to move to a ".old", or ".old.{timestamp}" than to remove it. You never quite know if there'll be a test failure that'll make you wish it was still there.

Reply:
To be clear, before this change it was overwritten (the next line copies over some existing common dir), so it was already lost. But that was extra buggy in the case where you previously had a 3-member network here and now have a 1-member network - at some point in recovery you try to use a `member_1.pem` that's left over and unrelated to this service.

The right fix, that I'll PR separately, is to put these into separate runners with their own labels so we stop overwriting interesting test history.